stock_auto_move/__init__.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf8 -*-
```

Comment:
Hi,
Thanks for your contribution.
Please use simple headers following OCA templates
https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py#L3